Source/Godunov.H

```diff
@@ -55,6 +70,33 @@ pc_transdo_passive(
   qm(ivp1, qc) = compo / rrnew;
 }
 
+AMREX_GPU_DEVICE
+AMREX_FORCE_INLINE
+void
+pc_transdo_linpassive(
```

Comment:
So I think this is close enough to `pc_transdo_passive` that we should be able to combine them. If you add `rr` to the list of inputs, then you can pass in 1 and 0 for `flxrho` and things should just work.

Reply:
So you think I should add two inputs for those representing `qyp(iv, QRHO)` and `qym(ivp1, QRHO)` to be different rr values?

Reply:
I don't think it's the same otherwise. Since it you'd end up with 
`rr qyp(iv,qc) / rr - compn / rr` instead of `qyp(iv,qc) - compn`

Reply:
The `compn` would end up being divided by rr.